pkg/mainflow.go

```diff
@@ -50,7 +50,7 @@ func Run(rawEnv map[string]string, namespace string) error {
 
 	printConfig(cfg)
 
-	l := launcher.New(checkup.New(), reporter.New(c, baseConfig.ConfigMapNamespace, baseConfig.ConfigMapName))
+	l := launcher.New(checkup.New(c, namespace, cfg), reporter.New(c, baseConfig.ConfigMapNamespace, baseConfig.ConfigMapName))
```

Comment:
why not use `baseConfig.ConfigMapNamespace`?

Reply:
The checkup creates its object in the namespace it lives in.
The ConfigMap object could theoretically come from another namespace (although we don't encourage it).